<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtUi</class>
 <widget class="QMainWindow" name="QtUi">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>216</height>
   </rect>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks</set>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="main_layout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="Header" name="header">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>MAP CONVERTER</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="Formatting">
        <property name="title">
         <string>Convert</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="b000">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;b&gt;Convert DirectX to OpenGL&lt;/b&gt;&lt;br&gt;
Converts a &lt;b&gt;Normal Map&lt;/b&gt; from &lt;b&gt;DirectX&lt;/b&gt; format to &lt;b&gt;OpenGL&lt;/b&gt; format.&lt;br&gt;&lt;br&gt;
&lt;b&gt;Usage:&lt;/b&gt; Select a &lt;b&gt;DirectX&lt;/b&gt; normal map.&lt;br&gt;&lt;br&gt;
- If an existing &lt;b&gt;OpenGL&lt;/b&gt; normal map of the same name is found, it will be &lt;b&gt;overwritten&lt;/b&gt;.</string>
           </property>
           <property name="text">
            <string>DirectX to GL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b001">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;b&gt;Convert OpenGL to DirectX&lt;/b&gt;&lt;br&gt;
Converts a &lt;b&gt;Normal Map&lt;/b&gt; from &lt;b&gt;OpenGL&lt;/b&gt; format to &lt;b&gt;DirectX&lt;/b&gt; format.&lt;br&gt;&lt;br&gt;
&lt;b&gt;Usage:&lt;/b&gt; Select an &lt;b&gt;OpenGL&lt;/b&gt; normal map.&lt;br&gt;&lt;br&gt;
- If an existing &lt;b&gt;DirectX&lt;/b&gt; normal map of the same name is found, it will be &lt;b&gt;overwritten&lt;/b&gt;.

</string>
           </property>
           <property name="text">
            <string>OpenGL to DX</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b002">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;b&gt;Pack Transparency into Albedo&lt;/b&gt;&lt;br&gt;
Combines a &lt;b&gt;Transparency (Opacity)&lt;/b&gt; map into the &lt;b&gt;Albedo&lt;/b&gt; map.&lt;br&gt;&lt;br&gt;
&lt;b&gt;Usage:&lt;/b&gt; Select an &lt;b&gt;Albedo&lt;/b&gt; and an &lt;b&gt;Opacity&lt;/b&gt; map.&lt;br&gt;&lt;br&gt;
- If an existing &lt;b&gt;Albedo Transparency&lt;/b&gt; map of the same name is found, it will be &lt;b&gt;overwritten&lt;/b&gt;.&lt;br&gt;
- If a &lt;b&gt;BaseColor&lt;/b&gt; map is provided, it will be converted to &lt;b&gt;Albedo&lt;/b&gt;.

</string>
           </property>
           <property name="text">
            <string>Albedo Transparency</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b003">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;b&gt;Pack Smoothness or Roughness into Metallic&lt;/b&gt;&lt;br&gt;
Combines a &lt;b&gt;Smoothness&lt;/b&gt; or &lt;b&gt;Roughness&lt;/b&gt; map into the &lt;b&gt;Metallic&lt;/b&gt; map.&lt;br&gt;&lt;br&gt;
&lt;b&gt;Usage:&lt;/b&gt; Select a &lt;b&gt;Roughness&lt;/b&gt; and a &lt;b&gt;Metallic&lt;/b&gt; map.&lt;br&gt;&lt;br&gt;
- If an existing &lt;b&gt;Metallic Smoothness&lt;/b&gt; map of the same name is found, it will be &lt;b&gt;overwritten&lt;/b&gt;.</string>
           </property>
           <property name="text">
            <string>Metallic Smoothness</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b004">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;b&gt;Convert Specular Glossiness to PBR&lt;/b&gt;&lt;br&gt;
Converts Specular Glossiness map(s) to &lt;b&gt;PBR&lt;/b&gt; format.&lt;br&gt;&lt;br&gt;
&lt;b&gt;Usage:&lt;/b&gt; Select &lt;b&gt;Diffuse, Specular,&lt;/b&gt; and &lt;b&gt;Glossiness&lt;/b&gt; maps.&lt;br&gt;&lt;br&gt;
- If no &lt;b&gt;Glossiness&lt;/b&gt; map is provided, it will attempt to extract it from the &lt;b&gt;Specular&lt;/b&gt; map.&lt;br&gt;
- If extraction fails, the &lt;b&gt;Roughness&lt;/b&gt; map will be approximated using the &lt;b&gt;Specular&lt;/b&gt; map.&lt;br&gt;
- If existing &lt;b&gt;BaseColor, Metallic,&lt;/b&gt; and &lt;b&gt;Roughness&lt;/b&gt; maps of the same name are found, they will be &lt;b&gt;overwritten&lt;/b&gt;.

</string>
           </property>
           <property name="text">
            <string>Spec Gloss to PBR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b006">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;b&gt;Optimize Texture&lt;/b&gt;&lt;br&gt;
Optimizes a texture by clamping the maximum size to &lt;b&gt;4K&lt;/b&gt; and adjusting the image 
(such as setting bit depth) according to the map type.&lt;br&gt;&lt;br&gt;
&lt;b&gt;Usage:&lt;/b&gt; Select the texture map(s) to optimize.&lt;br&gt;&lt;br&gt;
The optimized texture will be created in the same directory, 
with the original file moved to an &lt;i&gt;old&lt;/i&gt; folder.

</string>
           </property>
           <property name="text">
            <string>Optimize</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Header</class>
   <extends>QLabel</extends>
   <header>uitk.widgets.header.Header.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
